README.md

```diff
@@ -1,5 +1,3 @@
-Copyright (c) 2021 Western Digital Corporation or its affiliates.
```

Comment:
Arg, no. License and copyright are 2 different things. Adding the license file does not mandate remove the copryright. Please keep it.